docs/src/documentation/03-components/03-information/badge/03-accessibility.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/badge/accessibility/
+---
+
+## Accessibility
+
+The Badge component has been designed with accessibility in mind.
+
+### Screen reader support
+
+The Badge component supports screen readers through the `ariaLabel` prop. This prop is especially important when:
+
+- The badge contains only an icon
+- The visual content is not sufficient to convey the full meaning
```

Comment:
I would maybe skip the word `visual` here as screen readers don't care about _visual_ content, WDYT? I think there could be "Badge content" or just "content".